src/Api/Contact.php

```diff
@@ -13,38 +13,97 @@
 
 namespace Nexy\Gandi\Api;
 
+use fXmlRpc\Proxy;
+
 /**
  * @author Jérôme Pogeant <p-jerome@hotmail.fr>
+ *
```

Comment:
You may add your `@author` tag if you want. :+1:

Reply:
I'm not a big fan of `@author` annotation, no offense, but thank you anyway :laughing: